stuff/cron/plagiarism_detector.py

```diff
@@ -151,74 +186,92 @@ def filter_and_format_result(
     updated_result = []
     for result in results:
         updated_result.append(
-            Results(contest_id, 
-                    int(100*result[0]), # percentage match in file 1
-                    int(100*result[1]), # percentage match in file 2
-                    guid_and_submission_id_dict[result[2].split('/')[4].split('.')[0]], # file path 1 
-                    guid_and_submission_id_dict[result[3].split('/')[4].split('.')[0]], # file path 2
-                    json.dumps({'file1': get_range(result[4].split('\n')), # file info 1 -> range of lines
-                                'file2': get_range(result[5].split('\n'))}) # file info 2 -> range of lines
-                    )
-        )
-    # add to the database.     
+            Results(
+                contest_id=contest_id,
+                score_1=int(100 * result[0]),
+                score_2=int(100 * result[1]),
+                submission_id_1=guid_and_submission_id_dict[os.path.splitext(
+                    os.path.basename(result.test_filename))[0]],
+                submission_id_2=guid_and_submission_id_dict[os.path.splitext(
+                    os.path.basename(result.test_filename))[0]],
+                contents=json.dumps({
+                    'file1': get_range(result[4].split('\n')),
+                    'file2': get_range(result[5].split('\n'))
+                }),
+            ))
+    # add to the database.
     with dbconn.cursor() as cur:
         cur.executemany(INSERT_INTO_PLAGIARISMS, updated_result)
 
-def run_copy_detect(
-        dbconn: lib.db.Connection,
-        dir: str, contest_id: int,
-        submissions: Iterable[Mapping[str, Any]]) -> None:
-    
-    # we will run detector for each problem. 
+
+def run_copy_detect(dbconn: lib.db.Connection, dir: str, contest_id: int,
+                    submissions: Iterable[Submission]) -> None:
+
+    # we will run detector for each problem.
     for problem in os.listdir(dir):
-        detector = CopyDetector(test_dirs=
-                                    [(os.path.join(dir, problem))],
-                                    extensions=["cpp", "py", "py3", "java", "c"], display_t=0.9, autoopen = False, 
-                                    disable_filtering=True)
+        detector = copydetect.CopyDetector(
+            test_dirs=[(os.path.join(dir, problem))],
```

Comment:
```suggestion
            test_dirs=[os.path.join(dir, problem)],
```

there's a superfluous extra set of parens.

Reply:
Done.